test/unit/behaviors.html

```diff
@@ -127,6 +127,27 @@
     assert.equal(el.attributes.user.value, 'user', 'Behavior hostAttribute overrode user attribute');
   });
 
+  test('behaviour is null generates warning', function() {
+    var warned = false, oldWarn = Polymer.Base._warn;
+    Polymer.Base._warn = function (message) {
```

Comment:
It looks like `this._warn` is used for this warning, so let's just override the `_warn` of this element, if possible.